### ISSUE TITLE
issue: Duplicated attacks

### ISSUE THREAD

**Issue by Maxopoly** [tcgdex/cards-database#159]
### What seems to be the issue

{"label"=>"Card"}

### Field(s)

attacks

### Description

Some cards have a duplicated entry in 'attacks'. The duplicated entry seems to always have only a french translation. [Example](https://github.com/tcgdex/cards-database/blob/master/data/Sun%20%26%20Moon/Unbroken%20Bonds/1.ts#L106)

List of all cards which are seemingly affected by this issue, based on iterating them and searching for attacks without an english name:

```
bwp-BW85
dp1-120
dp1-121
dp1-122
dp3-10
dp3-83
dp3-86
dp3-95
dp5-4
dp5-11
dp5-57
dp5-72
dp5-78
dp5-91
hgss1-11
hgss1-30
hgss1-44
hgss1-46
hgss3-87
hgss3-88
hgss3-89
hgss3-90
hgss4-99
hgss4-100
hgss4-101
hgss4-102
sm5-58
sm5-62
sm5-77
sm5-89
sm6-31
sm6-74
sm6-96
sm7.5-7
sm8-19
sm8-144
sm8-154
sm9-10
sm9-51
sm10-1
sm10-20
sm10-107
sm10-130
sm10-191
sm10-192
sm10-194
sm11-35
sm11-54
sm11-71
sm11-126
sm11-214
sm11-215
sm11-217
sm11-218
sm11-220
sm11-221
sm11-222
sm11-225
sm11-226
sm12-210
sm12-211
sm12-212
sm12-214
sm12-215
sm12-216
sm12-223
sm115-29
sm115-44
smp-SM166
smp-SM168
smp-SM169
smp-SM201
xy1-114
xy2-13
xy2-69
xy2-79
xy2-107
xy2-108
xy2-109
xy5-55
xy5-149
xy5-151
xy6-35
xy6-59
xy6-61
xy6-76
xy6-100
xy6-102
xy6-105
xy7-8
xy7-11
xy7-15
xy7-18
xy7-21
xy7-28
xy7-32
xy7-35
xy7-43
xy7-50
xy7-67
xy7-85
xy7-92
xy7-96
xy7-97
xy7-98
xy8-12
xy8-50
xy8-79
xy8-92
xy8-104
xy8-113
xy9-18
xy9-41
xy10-19
xyp-XY39
xyp-XY63
xyp-XY87
xyp-XY154
```

**Comment by Maxopoly** [comment]
Some are also broken in more complicated ways,  for example https://api.tcgdex.net/v2/fr/cards/xy9-18 has as an attack what should be an ability. In the english API, this attack is sent as an empty JSON. Longterm it might be worthwhile to look into data validation through github actions as we expect certain field (combinations) to always exist.

 Also not sure if there are more, but https://assets.tcgdex.net/fr/xy/xy9/18/high.png is stretched.

**Comment by Maxopoly** [comment]
Or maybe as a followup to the VMAX rules always being the same, maybe TURBO evolution rules also shouldn't be listed as an attack?

**Comment by Aviortheking** [comment]
Those will be mostly fixed after I go through the database with my new(from the time I did those cards) scrapper

**Comment by Aviortheking** [comment]
I currently did the base set with it and I'm on the E-Card so it should be fixed in around 3-4 days

**Comment by Aviortheking** [comment]
Hey it should be fixed with #450 as the new German data made me fix those :D
